hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/common/table/read/TestHoodieFileGroupReaderOnSpark.scala

```diff
@@ -81,6 +81,7 @@ class TestHoodieFileGroupReaderOnSpark extends TestHoodieFileGroupReaderBase[Int
     sparkConf.set("spark.kryo.registrator", "org.apache.spark.HoodieSparkKryoRegistrar")
     sparkConf.set("spark.sql.extensions", "org.apache.spark.sql.hudi.HoodieSparkSessionExtension")
     sparkConf.set("spark.sql.parquet.enableVectorizedReader", "false")
+    sparkConf.set("spark.sql.orc.enableVectorizedReader", "false")
```

Comment:
the orc vec read is not supported well now? do we have separate tests for it?

Reply:
We don't support vectorized reads in FileGroupReader for spark yet. You'll see all the code paths for this manually disable the feature right now. It is something I want to explore in another branch to see if we can easily change this.

We do have tests that hit the vectorized reads and "returns batch" paths in spark for CoW in `TestHoodieMultipleBaseFileFormat`.